io.openems.edge.core/src/io/openems/edge/core/appmanager/translation_de.properties

```diff
@@ -164,6 +164,10 @@ App.PvInverter.Sma.phase.description = Angeschlossene Phase(n) des Wechselrichte
 App.PvInverter.SolarEdge.Name = SolarEdge PV-Wechselrichter
 
 # Time of use Tarif
+App.TimeOfUseTariff.controlMode.label = Kontroller Modus
+App.TimeOfUseTariff.controlMode.chargeConsumption = Aufladen der Batterie zu günstigen Preis
+App.TimeOfUseTariff.controlMode.delayDischarge = Verzögernung der Entladung während günstigen Preisen
```

Comment:
"Verzögerung", "günstiger Preise"


Please validate texts with a native German speaker (i.e. not Michael 😉 but Jan)